src/js/components/Ready/ReadyIntroduction.jsx

```diff
@@ -128,19 +132,20 @@ class ReadyIntroduction extends Component {
                   <StepText id="readyIntroductionStepText2">
                     Tell us what topics are important to you and we&apos;ll recommend people and organizations to follow as well as make ballot recommendations.
                     {' '}
-                    {voterIsSignedInWithTwitter ? (
-                      <>
-                        Since you are signed in with Twitter, you will see endorsements of everyone you follow on Twitter.
-                      </>
-                    ) : (
-                      <>
-                        <span className="u-link-color u-link-color-on-hover u-cursor--pointer" onClick={this.onSignInClick}>
-                          Link to your Twitter account
-                        </span>
-                        {' '}
-                        and see endorsements of everyone you follow on Twitter.
-                      </>
-                    )}
+                    { twitterEnabled && (
+                      voterIsSignedInWithTwitter ? 
```

Comment:
Thank you @Succorro, I believe you will need to add <> tags after line 135, and parenthesis after the "voterIsSignedInWithTwitter ?" line. Please run `npm run lint` to see if there are any AirBNB ES Lint errors with the files you have changed.

Reply:
Finished addressing the lint errors and commit the changes. Still getting accustomed to the application in addition to the workflow, if there are any mistakes I apologize in advance.